browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1288,11 +1288,15 @@ L.CanvasTileLayer = L.Layer.extend({
 			}
 			else if (textMsg.startsWith('a11ycaretchanged:')) {
 				obj = JSON.parse(textMsg.substring('a11yfocuschanged:'.length + 1));
-				this._map._textInput.onAccessibilityCaretChanged(parseInt(obj.position));
+				if (typeof this._map._textInput.onAccessibilityCaretChanged === 'function') {
```

Comment:
isn't `_textInput` class in 2 variants? 
- with a11y (and these functions)
- without a11y and I assume without these functions

What do you think about just adding these functions to the second implementation.
And put inside them some `console.error` - I assume it's bad we use a11y related code without using accessible textInput? would help debugging using logs

Reply:
Yes, it is a good idea.